src/main/java/appeng/core/sync/packets/PacketValueConfig.java

```diff
@@ -52,171 +51,168 @@
 import appeng.container.implementations.ContainerSecurity;
 import appeng.container.implementations.ContainerStorageBus;
 import appeng.core.sync.AppEngPacket;
-import appeng.core.sync.network.INetworkInfo;
+import appeng.core.sync.AppEngPacketHandler;
 import appeng.helpers.IMouseWheelItem;
 
 
-public class PacketValueConfig extends AppEngPacket
+public class PacketValueConfig implements AppEngPacket, AppEngPacketHandler<PacketValueConfig, AppEngPacket>
 {
 
-	public final String Name;
-	public final String Value;
+	private String name;
+	private String value;
 
 	// automatic.
-	public PacketValueConfig( ByteBuf stream ) throws IOException
+	public PacketValueConfig()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)